contract-map.json

```diff
@@ -3727,5 +3727,12 @@
     "erc20": true,
     "symbol": "XTM",
     "decimals": 18
+  },
+  "0xD33526068D116cE69F19A9ee46F0bd304F21A51f": {
+    "name": "Rocket Pool",
+    "logo": "rpl.png",
```

Comment:
SCG is required

Reply:
Unfortunately the icon uses some complex gradients and subtle shadows which don't scale well using an SVG, which is why we've opted with a png (we don't have SVGs available because of the gradient scaling).

Is there any flexibility around using a png?

Reply:
Have replaced with an SVG as requested.